shell/platform/windows/window_win32.cc

```diff
@@ -527,6 +529,17 @@ void WindowWin32::Destroy() {
 }
 
 void WindowWin32::HandleResize(UINT width, UINT height) {
```

Comment:
Instead of using the width and height values to determine the resize kind, we should check whether the message's `wParam` equals `SIZE_MINIMIZED`. See: https://docs.microsoft.com/en-us/windows/win32/winmsg/wm-size#parameters

```
#define SIZE_RESTORED       0
#define SIZE_MINIMIZED      1
#define SIZE_MAXIMIZED      2
#define SIZE_MAXSHOW        3
#define SIZE_MAXHIDE        4
```

Reply:
I found `WM_SHOWWINDOW`. I'll try later.
https://docs.microsoft.com/en-us/windows/win32/winmsg/wm-showwindow

Reply:
I also tried `WM_SHOWWINDOW` but the engine's window doesn't receive it.
For now I'll leave a comment in this line.

Reply:
Done, I added comments in Window::HandleResize.